runelite-client/src/main/java/net/runelite/client/config/ConfigManager.java

```diff
@@ -551,6 +565,15 @@ static String objectToString(Object object)
 			Keybind k = (Keybind) object;
 			return k.getKeyCode() + ":" + k.getModifiers();
 		}
+		if (object instanceof WorldPoint)
+		{
+			WorldPoint wp = (WorldPoint)object;
+			return wp.getX() + ":" + wp.getY() + ":" + wp.getPlane();
```

Comment:
Using `':'` (char) versus `":"` (string) will be more performant.

Reply:
That would be inconsistent with how we've done it for other classes before though